lib/pgBackRest/Backup/Backup.pm

```diff
@@ -852,13 +848,13 @@ sub process
         &log(TEST, TEST_BACKUP_RESUME);
 
         # Clean the old backup tmp path
-        $self->tmpClean($oFileLocal, $strBackupLabel, $oBackupManifest, $oAbortedManifest);
+        $self->tmpClean($oStorageRepo, $strBackupLabel, $oBackupManifest, $oAbortedManifest);
     }
     # Else create the backup path
     else
     {
         logDebugMisc($strOperation, "create backup path ${strBackupPath}");
-        $oFileLocal->pathCreate(PATH_BACKUP_CLUSTER, $strBackupLabel, undef, false, true);
+        $oStorageRepo->pathCreate(STORAGE_REPO_BACKUP . "/${strBackupLabel}");
```

Comment:
bCreateParent was made extraneous by the stanza-create function.  The parent directory must always exist or backup.info wouldn't.